tasks.rb

```diff
@@ -588,7 +588,7 @@ def get_values(osw_file, step)
     step.setArgument('skylight_area_right', 0)
     step.setArgument('skylight_ufactor', 0.33)
     step.setArgument('skylight_shgc', 0.45)
-    step.setArgument('door_area', 80.0)
+    step.setArgument('door_area', 40.0)
```

Comment:
I changed the sample files to have a more realistic amount of door area (40ft2 instead of 80ft2), so the OSWs also needed to change.